File: udemy/java/vet_media/src/vet_media2/Vet_media2.java
package vet_media2;
import javax.swing.JOptionPane;
public class Vet_media2 {
	public static void main(String[] args) {
		float notas[] = new float [5];
		float media;
		int cont, i;
		cont = 0;
		i = 0;
		media = 0;
		while (i <= 4) {
			notas[i] = Float.parseFloat(JOptionPane.showInputDialog("informe a nota do aluno numero "+(i+1)+": "));
			media = media + notas[i];
			if(notas[i]<6) {
				cont++;
			}
			i++;
		}
		media = media/5;
		JOptionPane.showMessageDialog(null,"A media dos alunos foi: "+media);
		if(cont != 0) {
			JOptionPane.showMessageDialog(null, cont+" alunos ficaram abaixo da media");
		}
	
	}

}

File: udemy/pascal/projetos/Vet_media2.pas
program Vet_media2;
var 
    notas : array[1..5] of real;
    cont , i : Integer;
    media : real;
begin
    cont :=0;
    media := 0;
    for i := 1 to 5 do
    begin
    writeln('informe a nota do ',i,'° aluno: ');
    read(notas[i]);
    media := media + notas[i];
    if(notas[i]<6) then
    cont := cont+1;
    end;
    media := media/5;
    writeln('A media dos alunos foi: ', media:3:2);
    if(cont > 0) then
    writeln(cont,' Alunos ficaram abaixo da media');
end.
